assets/admin/board_member_cpr_lookup.js

```diff
@@ -20,13 +20,25 @@ $(function () {
           $('#BoardMember_name').val('')
 
           // Indicate that identifier was not found
-          $($cprLookupButton).removeClass().addClass('btn-danger btn mt-2 lookup-cpr')
+          $($cprLookupButton).removeClass(function () {
+            const $regExp = /btn-[^\s]*/
```

Comment:
Prefixing variable names in JavaScript with `$` is usually only done with variables that hold a jQuery object, i.e. 

```js
const $el = $('div.name')
const name = $el.attr('name')
```

Reply:
Makes sense. Updated in 06eeac409da0163851f7c5fa41e925efaf384fcc.